scripts/agent_daemonset_old_version.yml

```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
```

Comment:
I am hesitant about maintaining two copies of this specification, how about keeping the `apps/v1` and guide users to modify the apiVersion using `sed` or `kustomize` if they are using an old version of kubernetes.